android/app/build.gradle

```diff
@@ -117,7 +117,12 @@ android {
         }
     }
     buildTypes {
+        debug {
+            // Fix for RN Debug on Android 8 - https://stackoverflow.com/questions/45940861/android-8-cleartext-http-traffic-not-permitted/50834600#50834600
+            manifestPlaceholders = [usesCleartextTraffic:"true"]
+        }
         release {
+            manifestPlaceholders = [usesCleartextTraffic:"true"]
```

Comment:
I've not done much Android or React Native development, but is it normal that a password manager needs to enable non-HTTPS communication? Do you know what plaintext HTTP resources this app requests?

Reply:
Good find @jokeyrhyme! Cleartext was needed to allow access to the metro bundler in dev - I must have forgotten to switch it back to off for release.

This is fixed in feccc8a